README.md

```diff
@@ -50,3 +50,33 @@ t.setAttribute("PluginName", "ExamplePlugin")
 ## Setting an Application Level Plugin
 
 In the config set `Webapi.app.plugins` to your plugin name. This can be a comma separated list to support multiple plugins.
+
+## Login Plugin
+
+A Login Plugin loads before the user log into the Web Client, and it allows UI permissions to be set. See `ExampleLoginPlugin.jsx` for an example.
+The current supported permissions are:
+
+- canUsePanels - Allows the user access to the `Panels` button in the UI.
+- canCreateDashboard - Allows the user to create a new Dashboard.
+- canCreateCodeStudio - Allows the user to create a new Code Studio.
+- canCreateQueryMonitor - Allows the user to open the Query Monitor.
```

Comment:
May as well keep the language consistent
```suggestion
- canCreateQueryMonitor - Allows the user to create a new Query Monitor.
```

Reply:
Fixed